ref_arch/real_time_dashboards.rst

```diff
@@ -139,14 +139,14 @@ This looks a lot like the previous code block. Most importantly: It also shards
 replication factor. Because all three of those match, there's a 1-to-1
 correspondence between ``http_request`` shards and ``http_request_1min`` shards,
 and Citus will place matching shards on the same worker. This is called
-colocation; it makes queries such as joins faster and our rollups possible.
+co-location; it makes queries such as joins faster and our rollups possible.
```

Comment:
How about I make a note to open a PR which does this uniformly throughout the docs. Not every single occurrence of the word co-location of course, but representative ones from each section. It'll probably be smoother to do this once the current PRs are merged so it doesn't make bogus little git conflicts.